pyproject.toml

```diff
@@ -19,3 +54,17 @@ target-version = "py37"
 
 [tool.ruff.mccabe]
 max-complexity = 10
+
+[tool.setuptools]
+include-package-data = false
+license-files = ["LICENSE"]
+packages = [
+  "social_django",
+  "social_django.migrations",
+  "social_django.management",
+  "social_django.management.commands"
+]
+zip-safe = false
+
+[tool.setuptools.dynamic]
+version = {attr = "social_django.__version__"}
```

Comment:
Does this work for you? I tried this and get a error from `pip install -e .` looks like:
```
        File "/tmp/pip-build-env-rd1a7s2m/overlay/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py", line 126, in read_configuration
          validate(subset, filepath)
        File "/tmp/pip-build-env-rd1a7s2m/overlay/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py", line 51, in validate
          raise ValueError(f"{error}\n{summary}") from None
      ValueError: invalid pyproject.toml config: `project`.
      configuration error: `project` must contain ['version'] properties
```